src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/EventService.java

```diff
@@ -397,6 +400,8 @@ public void processEvents() {
                   collectionExerciseService.transitionCollectionExercise(
                       event.getCollectionExercise(),
                       CollectionExerciseDTO.CollectionExerciseEvent.GO_LIVE);
+                  sessionResolver.removeActiveSession(
+                      String.valueOf(event.getCollectionExercise().getSurveyId()));
```

Comment:
What happens in the error state for this by the way?  I think as it stands, it would stop everything because it would throw an exception, bubble up and just raise a 500 for the cronjob to ignore.

I don't think deleting the key should be a showstopper as it naturally will go away in a maximum of 10 minutes.  You should catch whatever exception it throws, log an error and carry on like all the other statements in this function generally do.